exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -197,16 +197,19 @@ const NINE_POINT_SCALE = TEN_POINT_SCALE.slice(0, 10);
 
 var generateValidators = function (questions) {
     var validators = {};
-    var presence = validator('presence', {
-        presence: true,
-        message: 'This field is required'
-    });
     for (var i = 0; i < questions.length; i++) {
         for (var item = 0; item < questions[i]['items'].length; item++) {
             var isOptional = questions[i]['items'][item].optional;
             if (!isOptional) {
                 var key = `questions.${i}.items.${item}.value`;
-                validators[key] = presence;
+                validators[key] = validator('presence', {
+                    presence: true,
+                    message: 'This field is required',
```

Comment:
This message actually doesn't get shown, since the continue button is disabled until all required questions have been answered.